spec/models/claim/base_claim_spec.rb

```diff
@@ -704,6 +704,18 @@ def make_date_params(date_string)
   }
 end
 
+describe '#agfs_reform?' do
+  let(:claim) { MockBaseClaim.new }
+  let(:mock_fee_scheme) { double(:fee_scheme) }
+
+  specify {
+    expect(FeeScheme).to receive(:for_claim).with(claim).and_return(mock_fee_scheme)
+    expect(mock_fee_scheme).to receive(:agfs_reform?)
+    claim.agfs_reform?
```

Comment:
These tend to be a bit of dumb specs, but to complement the already existent expectations, consider to assert that the actual return value of `claim.agfs_reform?` is the same as the one returned by `mock_fee_scheme.agfs_reform?`, as the method in the claim could be something like this:

```ruby
def agfs_reform?
  fee_reform.agfs_reform?
  return 'something else'
end
```

(which is not the case, but it's just to illustrate why is probably better to assert that. 

I was actually looking for some RSpec expectations that just said something like:

```
# meta code :P
expect(claim).to_delegate(:agfs_reform?).to(:fee_scheme)
```

Reply:
because we have gem 'shoulda-matchers', we can, and do, use  `it { should delegate_method(:meth).to(:object) }`